3rdParty/velocypack/src/Dumper.cpp

```diff
@@ -376,7 +376,7 @@ void Dumper::dumpValue(Slice const* slice, Slice const* base) {
     }
 
     case ValueType::Object: {
-      ObjectIterator it(*slice);
+      ObjectIterator it(*slice, true);
```

Comment:
This could have huge consequences in lots of places. I do not know if we can do this...

Reply:
Also, if we do this here, we should also do it in the VelocyPack repository...

Reply:
reverted that change in 3.3, added it to the changelog >= 3.4